postman/Find Subscribers Across Lists.postman_collection.json

```diff
@@ -0,0 +1,164 @@
+{
+	"info": {
+		"_postman_id": "a3f9d2bf-beea-4df2-84d7-b11576abd0a3",
+		"name": "Find Subscribers Across Lists",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Get Account",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "74e7dce1-167a-4675-8431-bdf2223fcd25",
+						"exec": [
+							"pm.test(\"Successfully get an account Id\", function(){",
+							"    pm.response.to.have.status(200);",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.entries[0].id).to.be.a('number')",
+							"    pm.globals.set(\"account_id\",jsonData.entries[0].id )    ",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_URL}}/1.0/accounts",
+					"host": [
+						"{{base_URL}}"
+					],
+					"path": [
+						"1.0",
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Lists",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "f6163948-14a0-48e5-ad18-ee37ab1a3c0d",
+						"exec": [
+							"pm.test(\"Successfully searched for subscribers across all lists\", function() {",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_URL}}/accounts/{{account_id}}/?ws.op=findSubscribers&email={{subscriber_email}}",
+					"host": [
+						"{{base_URL}}"
+					],
+					"path": [
```

Comment:
This path doesn't include the 1.0, but the one above does. In either case, I suggest removing the environment variable.